packages/web-frontend/src/screens/mobile/HomeScreen/index.js

```diff
@@ -16,7 +15,7 @@ import { Dashboard } from '../../../components/mobile/Dashboard';
 import { setOrgUnit, toggleDashboardSelectExpand, setDashboardGroup } from '../../../actions';
 import { WHITE } from '../../../styles';
 import {
-  selectCurrentDashboardGroupCode,
+  selectCurrentDashboardName,
```

Comment:
So before, in a lot of places, the variable/method names are often `dashboardGroupCode`, even though they are `dashboardName` instead of codes. I found it a bit confusing. So I tried to change the URL to used `dashboardCode` instead of `dashboardName`, but this is tricky because in a lot of places we only have access to `dashboardName` (eg: project default project is reference by `dashboardName`). So I went with changing the variable/method names from `dashboardGroupCode` to `dashboardName` instead to avoid confusion